src/main/webapp/dataset.xhtml

```diff
@@ -86,7 +86,7 @@
                     <meta property="og:title" content="#{DatasetPage.title}" />
                     <meta property="og:type" content="article" />
                     <meta property="og:url" content="#{DatasetPage.dataverseSiteUrl}/dataset.xhtml?persistentId=#{dataset.globalId}" />
-                    <meta property="og:image" content="#{DatasetPage.dataset.getDatasetThumbnail(ImageThumbConverter.DEFAULT_PREVIEW_SIZE) == null ? DatasetPage.dataverseSiteUrl.concat(resource['images/dataverse-icon-1200.png']): DatasetPage.dataverseSiteUrl.concat('/api/datasets/:persistentId/thumbnail?persistentId=').concat(DatasetPage.dataset.getGlobalId().asString())}" />
+                    <meta property="og:image" content="#{empty DatasetPage.thumbnailString ? DatasetPage.dataverseSiteUrl.concat(resource['images/dataverse-icon-1200.png']): DatasetPage.dataverseSiteUrl.concat('/api/datasets/:persistentId/thumbnail?persistentId=').concat(DatasetPage.dataset.getGlobalId().asString())}" />
```

Comment:
Note: Part of #10687 was a different fix for this line - looks like it was asking for a size 0 preview. This fix here is fine - neither are actually asking for the same size that the api call returns (ImageThumbConverter.DEFAULT_CARDIMAGE_SIZE), so using a cached value makes sense.

Reply:
Oh wow, I completely missed all that - that you already had an issue and a PR for the bug in question. :(

Reply:
But yes, my logic was that if we have a 140px thumbnail available, generating the tiny one should not be a problem either. 
I will add a test for `if (size == 0) { ... return null; }` to DatasetUtil.java though.

Reply:
That's in the other PR I think

Reply:
That's what I meant, that I was going to copy-and-paste these lines from #10687; done. 
Same for the release note, added the recommendations to use the Clear apis, decided not to mention the direct database query.